api/v1/flow/README.md

```diff
@@ -267,6 +267,7 @@ EventTypeFilter is a filter describing a particular event type
 | uuid | [string](#string) |  | uuid is a universally unique identifier for this flow. |
 | verdict | [Verdict](#flow-Verdict) |  |  |
 | drop_reason | [uint32](#uint32) |  | **Deprecated.** only applicable to Verdict = DROPPED. deprecated in favor of drop_reason_desc. |
+| auth_type | [string](#string) |  | auth_type is the authentication type specified for the flow in Cilium Network Policy. Empty when authentication is not required. |
```

Comment:
nit: Maybe document that this is only set for policy verdict events

Reply:
Will do.

Reply:
Done.